.gitignore

```diff
@@ -3,7 +3,8 @@ build/
 tutorials/rst-tutorials/
 tutorials/notebooks/FITS-cubes/Maps/
 IPython-*
-*.fits
+*.fits*
+*.gz
```

Comment:
Do you still need `*.fits*` if `.fits.gz` is going to be caught by `*.gz` anyway?

Reply:
🤷‍♂ why not?

Reply:
https://www.youtube.com/watch?v=OawrlVoQqSs -- Okay, sure.